src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php

```diff
@@ -26,11 +26,25 @@ class SymfonyTestsListener extends \PHPUnit_Framework_BaseTestListener
     private $wasSkipped = array();
     private $isSkipped = array();
 
-    public function __construct(array $extraClockMockedNamespaces = array())
+    public function __construct(array $mockedNamespaces = array())
```

Comment:
I think we should now document which keys (groups) are supported by the listener.